example/client/app/app.go

```diff
@@ -33,6 +34,10 @@ func main() {
 	port := os.Getenv("PORT")
 	scopes := strings.Split(os.Getenv("SCOPES"), " ")
 	responseMode := os.Getenv("RESPONSE_MODE")
+	pkce, err := strconv.ParseBool(os.Getenv("PKCE"))
+	if err != nil {
+		logrus.Fatalf("error parsing PKCE %s", err.Error())
+	}
 
```

Comment:
When `PKCE` is not set, the example fails with:

```
FATA[0000] error parsing PKCE strconv.ParseBool: parsing "": invalid syntax 
exit status 1
```

The unset variant needs to be handled first. 

```suggestion

	var pkce bool
	if pkceEnv, ok := os.LookupEnv("PKCE"); ok {
		var err error
		pkce, err = strconv.ParseBool(pkceEnv)
		if err != nil {
			logrus.Fatalf("error parsing PKCE %s", err.Error())
		}
	}
```

Reply:
I overlooked that! Thanks for pointing it out—I’ve incorporated your suggestion.